lib/rules/no-magic-numbers.js

```diff
@@ -90,6 +95,18 @@ module.exports = {
             return ignore.indexOf(value) !== -1;
         }
 
+        /**
+         * Returns whether the number should be ignored when used as a default
+         * value assignment.
+         * @param {ASTNode} fullNumberNode `Literal` or `UnaryExpression` full number node
+         * @returns {boolean} true if the number should be ignored
+         */
+        function isIgnoredDefaultValue(fullNumberNode) {
```

Comment:
Just a small change: the name should be `isDefaultValue`.

The description of the function and description of its return value should be also updated since the function now doesn't check if the number should be ignored (it just checks if it's a default value).

Reply:
Done.